src/components/Octocorp.tsx

```diff
@@ -0,0 +1,55 @@
+import { BodyText, HeaderText, PrimaryAccentText } from 'components/Text'
+import Button from 'components/Button'
+import Octocorp from 'icons/OctocorpLogo'
+import classnames, {
+  alignItems,
+  display,
+  flexDirection,
+  margin,
+  maxWidth,
+  space,
+} from 'classnames/tailwind'
+
+const textCard = classnames(space('space-y-2'), maxWidth('max-w-400'))
+const textCardHeader = space('space-y-4')
+const textCardBody = space('space-y-6')
+const wrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col', 'lg:flex-row'),
+  alignItems('items-center'),
+  space('space-y-40'),
+  margin('mb-72')
+)
+
+export default function () {
```

Comment:
Did you check it on screen with `280px` size? Is everything okay?

Reply:
I don't think we want to use very big `margin` or `padding`. If everything can be handled by `space` would be optimal but using alternatives are fine as well such as `height`